### DISCUSSION TITLE
Couldn't see the contents of ipynb file

### DISCUSSION THREAD

**Discussion by Anesha-Nagarajan** [community/community#124611] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I couldn't see the contents of python notebook in Git Hub

**Comment by CopperEagle** [comment]
Hey @Anesha-Nagarajan 

This affects the whole page, see #124617. It is an issue on their side with nbconvert. May need to wait until tomorrow for a fix. 

Hope this helps

**Comment by davevad93** [comment]
Hi @Anesha-Nagarajan ,

If you need to see the content of your `ipynb` files you could use [nbviewer](https://nbviewer.org/) (only for public repos) or [ReviewNB](https://www.reviewnb.com/?utm_source=reviewnb_blog) (also for private repos).

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Anesha-Nagarajan ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
